tests/integration/cmd_dev_debug_test.go

```diff
@@ -122,6 +122,72 @@ var _ = Describe("odo dev debug command tests", func() {
 			})
 		})
 	})
+	FWhen("a composite apply command is used as debug command", func() {
+		deploymentName := "my-component"
+		var session helper.DevSession
+		var sessionOut []byte
+		var err error
+		var ports map[string]string
+		const (
+			DEVFILE_DEBUG_PORT = "5858"
+		)
+
+		BeforeEach(func() {
+			helper.CopyExample(filepath.Join("source", "devfiles", "nodejs", "project"), commonVar.Context)
+			helper.CopyExampleDevFile(filepath.Join("source", "devfiles", "nodejs", "devfile-composite-apply-commands-debug.yaml"), filepath.Join(commonVar.Context, "devfile.yaml"))
```

Comment:
nit: `devfile-composite-apply-commands-debug.yaml` and `devfile-composite-apply-commands.yaml` look quite similar. Maybe you could just add the debug command (and endpoint) to `devfile-composite-apply-commands-debug.yaml` and use it here as well?

Reply:
Definitely not a nit! That's a good suggestion :smile: